### PR TITLE
Add Serbian translations for types_strings per @ZeljkoBG discussions/9858

### PR DESCRIPTION
**Pull request by zyphlar**

### CODE REVIEW

**Comment by zyphlar** [comment]
@ZeljkoBG @biodranik @dobridabar here is a types_string translation. Since this file doesn't exist in Serbian yet it should have no conflicts and only `sr = ` additions.

**Comment by dobridabar** [comment]
Kudos to @ZeljkoBG. You did a great job. I just find some small mistakes, I hope you will accept them as a constructive.

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -158,6 +162,7 @@
     pt-BR = Telesquis
     ru = Бугельная канатная дорога
     sk = Vlek
+    sr = Ски лифт једносед
```
[the remark this answers is not in the document]
Comment (ZeljkoBG) [reply data/strings/types_strings.txt:165]: Let there be no confusion. If I need to approve something, I don't know how to do it. I'm afraid of making a mistake.
github is not my go-to place. And it's not like other platforms.

[the remark this answers is not in the document]
Comment (zyphlar) [reply data/strings/types_strings.txt]: @ZeljkoBG it's ok, the suggestion and approval process is for the pull request opener. If you click the Edit button in the future it'll be you but for now it's me.

**Review by ZeljkoBG — APPROVED** [review]

**Comment by zyphlar** [comment]
@ZeljkoBG @dobridabar you should be able to download the WebDebug APK here and install it on your Android phone to test these translations:

https://github.com/organicmaps/organicmaps/actions/runs/12360244019?pr=9871

**Review by pastk — APPROVED** [review]

**Comment by ZeljkoBG** [comment]
> @ZeljkoBG @dobridabar you should be able to download the WebDebug APK here and install it on your Android phone to test these translations:
> 
> https://github.com/organicmaps/organicmaps/actions/runs/12360244019?pr=9871

Uuuuu jedva sam se snašao. Instalirao sam.

**Comment by ZeljkoBG** [comment]
@zyphlar i need help
When I go to correct the error, on the page
https://github.com/organicmaps/organicmaps/pull/9869/files/2b2f6a6f90c7cb1ee3c5f52b09569b482221a639
there are two translations, left and right.

Which translation am I correcting? Mine on the left?

**Comment by dobridabar** [comment]
@ZeljkoBG, why did you close this PR? Was it by accident?

My understanding was that we are closing #9869.

**Comment by zyphlar** [comment]
Yeah this one needs to stay open.

@ZeljkoBG since this hasn't been merged yet the best thing to do is simply tell me the incorrect and correct versions of a string and I can make the change. In the future when it's your pull request you can probably do it with the Edit button. 

In GitHub, green + is new and red - is old.

**Comment by ZeljkoBG** [comment]
ok

**Comment by dobridabar** [comment]
> @ZeljkoBG @dobridabar you should be able to download the WebDebug APK here and install it on your Android phone to test these translations:

@zyphlar, I installed and tested. Looks good, but I would like to make some more changes. What should I do next? Should I propose them on this PR, or should I do it in a subsequent one? I already proposed more then 50 changes on this PR, and I don't know how are we going to resolve them. Plus I would like this PR to be merged before next release, if possible, even if not perfect.

**Comment by zyphlar** [comment]
@dobridabar I'll let you judge. I've added you as a collaborator to my repo so you could push changes to zyphlar/serbian-types and affect this PR if you want. I could also apply any patch files that you want to send over within a day or two of you sending. You could also @ a core maintainer to request this be merged (I'm not authorized to merge.)

**Pull request merged**